### PR TITLE
Added root cert and installation steps to Dockerfile

### PR DESCRIPTION
**Pull request by ocraviotto**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,9 @@
 FROM debian:jessie
 
+COPY FWDRooCA.pem /tmp/FWDRooCA.crt
```
Comment (jpalomaki) [inline Dockerfile]: I presume this ought to be `FWDRootCA.pem`? I.e. root over roo

Comment (ocraviotto) [inline Dockerfile]: Yep, that was the idea. Good I copied-pasted every time, and checked the image builds, but will change anyway though.

**Comment by ibiris** [comment]
LGTM :+1:

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,9 @@
 FROM debian:jessie
 
+COPY FWDRootCA.pem /tmp/FWDRootCA.crt
```
Comment (ilbambino) [inline Dockerfile]: not that is that important, but why to copy to a temp location, and not to the `/usr/local/share/ca-certificates/` directly?

Comment (ocraviotto) [inline Dockerfile]: Because (AFAIR) the share/ca-certificates/ does not exist before installing ca-certificates.

**Pull request merged**